src/next-app/components/NavbarData.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import * as IoIcons from 'react-icons/io';
+
+export const NavbarData = [
+  {
+    title: 'Home',
+    path: '/dashboard',
+    icon: <IoIcons.IoMdHome />,
+  },
+  {
+    title: 'Services',
+    path: '#',
+    icon: <IoIcons.IoMdConstruct />,
+  },
+  {
+    title: 'Appointments',
+    path: '#',
+    icon: <IoIcons.IoMdCalendar />,
+  },
+  {
+    title: 'Employees',
+    path: '/invite',
+    icon: <IoIcons.IoMdPeople />,
+  },
+  {
+    title: 'Work Orders',
+    path: '#',
+    icon: <IoIcons.IoMdCalculator />,
+  },
+  {
+    title: 'Quotes',
+    path: '#',
+    icon: <IoIcons.IoMdFiling />,
+  },
+];
```

Comment:
This file is not a component, and so shouldn't be in the components directory. I recommend creating a "constants" directory and putting it in there.